Modules/Loadable/Terminologies/Logic/vtkSlicerTerminologiesModuleLogic.h

```diff
@@ -269,6 +271,13 @@ class VTK_SLICER_TERMINOLOGIES_LOGIC_EXPORT vtkSlicerTerminologiesModuleLogic :
   /// Assemble human readable info string from a terminology entry, for example for tooltips
   static std::string GetInfoStringFromTerminologyEntry(vtkSlicerTerminologyEntry* entry);
 
+  /// Compare two terminology entries for equality.
+  /// \return True if the entries are equal, false otherwise.
```

Comment:
Put documentation group markers `//@{` and `//@}` around this section, otherwise the documentation is only assigned to the first method.